docs/Operators.md

```diff
@@ -12744,30 +12744,30 @@ This version of the operator has been available since version 10 of the default
 #### Inputs (8 - 9)
 
 <dl>
-<dt><tt>x</tt> : T1</dt>
+<dt><tt>x</tt> (differentiable) : T1</dt>
```

Comment:
This is differentiable but in `ConvInteger` the input `X` isn't.

Reply:
Removed the differentiable variables for QlinearConv and ConvInteger operators